src/storages/pluggable/constants.ts

```diff
@@ -0,0 +1 @@
+export const logPrefix = 'storage:pluggable:';
```

Comment:
Suggestion that I'm not sure if we are following everywhere: an usual convention for managing constants is to write them `UPPER_CASE` (and snake_case)